### DISCUSSION TITLE
Is it possible to change the texture color of rendering glb model from Android using explicit Intent in Scene/AR viewer?

### DISCUSSION THREAD

**Discussion by ghost** [google/model-viewer#2500] [Q&A]
I m wondering to change the rendering object color  within AR/Scene viewer. I have done rendering using Android explicit Intent. Basically, I want to add few color buttons on ar view and on click of any button the rendering object texture color should change.

Thank in advance.

**Comment by elalish** [comment]
No, only in WebXR. This is part of why this is our default AR mode now.
